aixplain/factories/finetune_factory/__init__.py

```diff
@@ -57,9 +60,55 @@ def _create_cost_from_response(cls, response: Dict) -> FinetuneCost:
         """
         return FinetuneCost(response["trainingCost"], response["inferenceCost"], response["hostingCost"])
 
+    @classmethod
+    def _get_data_list(cls, dataset: Dataset):
+        flatten_target_values = [item for sublist in list(dataset.target_data.values()) for item in sublist]
+        data_list = list(dataset.source_data.values()) + flatten_target_values
+        return data_list
+
+    @classmethod
+    def _validate_prompt(cls, prompt: Text, dataset_list: List[Dataset]) -> Text:
```

Comment:
Do not make this a classmethod. Create a separate script for this method together with the previous `_get_data_list`.